src/main/java/com/devathon/preguntonic/config/WebSocketConfig.java

```diff
@@ -37,9 +37,8 @@ public void registerStompEndpoints(StompEndpointRegistry registry) {
   @Override
   public void configureMessageBroker(MessageBrokerRegistry registry) {
     registry
-        .setApplicationDestinationPrefixes("/app") // [1]The prefix for the application destination.
-        .enableSimpleBroker(
-            "/room"); // [2]Enable the simple broker (some examples of destinations).
+        .setApplicationDestinationPrefixes("/preguntonic") // [1]The prefix for the application destination.
+        .enableSimpleBroker("/"); // [2]Enable the simple broker (some examples of destinations).
```

Comment:
Why is this change in the broker suggested? 
if it is needed for more `topics` isn't it possible to do something like: 

```suggestion
        .setApplicationDestinationPrefixes("/preguntonic") // [1]The prefix for the application destination.
        .enableSimpleBroker("/room", "/chat", "/topic"); // [2]Enable the simple broker (some examples of destinations).
```

Reply:
anyway this also affect the other controller... but depending of which PR is merge first we can resolve conflicts

Reply:
We only need `/rooms` broker, so we can use that in all topics